api.md

```diff
@@ -39,10 +39,10 @@
     ```
 
 * `/api/institutions/<institution>/periods/<period>/submissions/<submission>/edits`
-    * `GET` - List of all edits for a given submission, grouped by edit type. Structure: `{"syntactical": {...}, "validity": {..}, "quality": {...}, "macro": {...}}`
+    * `GET` - List of all edits for a given submission, grouped by edit type. Structure: `{"syntactical": {...}, "validity": {..}, "quality": {...}, "macro": {...}, "q029": {...}, "q595": {...}}`
     * `PUT` - Bulk update of edits, used for verification - TODO
 
-* `/api/institutions/<institution>/periods/<period>/submissions/<submission>/edits/syntactial|validity|quality|macro`
+* `/api/institutions/<institution>/periods/<period>/submissions/<submission>/edits/syntactial|validity|quality|macro|q029|q595`
```

Comment:
First two bullets:
Querying for Q029 edits would involve hitting the endpoint at `.../edits/q029` while getting all quality edits would be `.../edits?type=quality`. Quality is a filter and the individual edits are members of the hierarchy.

re: lar key, this might be the wrong track, as we'll need to support PUTs on individual edits/lars see #138. So perhaps something more structured will be needed. `.../edits/<edit>/lars/<lar>`